call_server/static/scripts/site/search.js

```diff
@@ -74,15 +74,15 @@
         });
 
       } else {
-        // hit Sunlight OpenStates
+        // hit OpenStates
 
         // TODO, request state metadata
         // display latest_json_date to user
 
         $.ajax({
-          url: CallPower.Config.SUNLIGHT_STATES_URL,
+          url: CallPower.Config.OPENSTATES_URL,
           data: {
-            apikey: CallPower.Config.SUNLIGHT_API_KEY,
+            apikey: CallPower.Config.OPENSTATES_API_KEY,
             state: campaign_state,
```

Comment:
To work with the [v3 api](https://v3.openstates.org/docs#/people/people_search_people_get) it looks like some of these params need to change

`state` -- became `jurisdiction`

`last_name` -- became just `name`

`in_office` -- seems to not be supported at all anymore

`chamber` -- became `org_classification` and should be one of "legislature", "executive", "lower", "upper", "government"